scheduler/docs/scheduler-rest-api.asc

```diff
@@ -69,6 +69,7 @@ The request body is list of job maps with each entry containing a map of the fol
 |JSON Key | Type | Description
 |`uuid` | string | a user provided UUID for tracking and referring to the job.
 |`command` | string | the command to run
+|`cook_executor` | boolean | Flag to force use of the Cook executor, defaults to false.
```

Comment:
Ah, I see. Should we just latch onto that existing flag, and allow it to be `cook|custom|mesos`?